src/quorum/model/yaml/docker-compose/validatorDockerComposeYaml.ts

```diff
@@ -20,7 +20,7 @@ class ValidatorDockerComposeYaml extends DockerComposeYaml {
       volumes: [`${bdkPath}/validator${validatorNum}/data/:/data`],
       entrypoint: [
         '/bin/sh', '-c',
-        `geth init --datadir /data /data/genesis.json; geth --datadir /data --networkid ${chainId} --nodiscover --verbosity 3 --syncmode full --nousb --mine --miner.threads 1 --miner.gasprice 0 --emitcheckpoints --http --http.addr 0.0.0.0 --http.port 8545 --http.corsdomain "*" --http.vhosts "*" --ws --ws.addr 0.0.0.0 --ws.port 8546 --ws.origins "*" --http.api admin,trace,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,istanbul,qbft --ws.api admin,trace,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,istanbul,qbft --port ${peerPort} `,
+        `geth init --datadir /data /data/genesis.json; geth --datadir /data --networkid ${chainId} --verbosity 3 --syncmode full --nousb --mine --miner.threads 1 --miner.gasprice 0 --emitcheckpoints --http --http.addr 0.0.0.0 --http.port 8545 --http.corsdomain "*" ${(bootnode) ? '--bootnodes '.concat(nodeEncode) : ''} --http.vhosts "*" --ws --ws.addr 0.0.0.0 --ws.port 8546 --ws.origins "*" --http.api admin,trace,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,istanbul,qbft --ws.api admin,trace,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,istanbul,qbft --port ${peerPort} `,
```

Comment:
after delete `nodiscover`, is it still comply with static node standard?

and we should make this entrypoint more beautiful

Reply:
https://docs.goquorum.consensys.io/configure-and-manage/configure/static-nodes#configure-static-nodes-1

Reply:
${(bootnode) ? '--bootnodes '.concat(nodeEncode) : '--nodiscover'}